src/screen/CreateUserScreen.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { Button, Input } from 'react-native-elements';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import Block from '../components/Block';
+import { regNewUser } from '../actions/UserActions';
+
+const CreateUserScreen = ({
+  navigation,
+  createUser,
+}) => {
+  const { t } = useTranslation();
+
+  const [userName, setUserName] = useState();
+
+  const [userPass, setUserPass] = useState();
+
+  return (
+    <Block height width>
+      <Input
+        placeholder={t('userName')}
+        value={userName}
+        onChangeText={value => setUserName(value)}
+      />
+      <Input
+        placeholder={t('userPass')}
+        value={userPass}
+        onChangeText={value => setUserPass(value)}
```

Comment:
See this video 
https://www.youtube.com/watch?time_continue=1413&v=dpw9EHDh2bM
49:55 
Dan Abramov wrote custom hook for Input.